modules/dpu-enable-dpu-hw-offloading.adoc

```diff
@@ -0,0 +1,330 @@
+
+.. Save the YAML in the `SriovNetworkNodePolicy.yaml` file:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: policy-mlnx-bf <1>
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: mlnx_bf <2>
+  nodeSelector:
+    node-role.kubernetes.io/dpu-host: <3>
+  priority: 99 <4>
+  numVfs: 4 <5>
+  nicSelector: <6>
+    vendor: "15b3" <7>
+    deviceId: "a2d6" <8> 
+    pfNames: ['ens1f0'] <9>
+    rootDevices: ['0000:3b:00.0'] <10>
+----
++
+<1> The name for the custom resource object.
++
+<2> The resource name of the SR-IOV network device plug-in. You can create multiple SR-IOV network node policies for a resource name.
++
+<3> The node selector specifies the nodes to configure. Only SR-IOV network devices on the selected nodes are configured. The SR-IOV Container Network Interface (CNI) plug-in and device plug-in are deployed on selected nodes only.
++
+<4> Optional: The priority is an integer value between 0 and 99. A smaller value receives higher priority. For example, a priority of 10 is a higher priority than 99. The default value is 99.
++
+<5> The number of the virtual functions (VF) to create for the SR-IOV physical network device. For a Mellanox NIC, the number of VFs cannot be larger than 128.
++
+<6> The NIC selector identifies the device for the Operator to configure. You do not have to specify values for all the parameters. It is recommended to identify the network device with enough precision to avoid selecting a device unintentionally.
++
+<7> The vendor hexadecimal code of the SR-IOV network device. Vendor id `15b3` is for Mellanox devices.
++
+<8> The device hexadecimal code of the SR-IOV network device. For example, `a2d6` is the device ID for a Bluefield-2 DPU device.
++
+<9> An array of one or more physical function (PF) names for the device.
++
+<10> An array of one or more PCI bus addresses for the PF of the device. Provide the address in the following format: `0000:02:00.1`.
++
+.. Create the SriovNetworkNodePolicy object:
++
+[source,terminal]
+----
+$ oc create -f SriovNetworkNodePolicy.yaml
+----
+
+. Create a Cluster Network Operator (CNO) ConfigMap in the tenant cluster setting the mode to `dpu-host`. Save the YAML in the `sriovdpuconfigmap.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: dpu-mode-config
+    namespace: openshift-network-operator
+data:
+    mode: "dpu-host"
+immutable: true
+----
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ oc create -f sriovdpuconfigmap.yaml.yaml
+----
+
+. Set the environment variable `OVNKUBE_NODE_MGMT_PORT_NETDEV` for each DPU host. Save the YAML in the `setenvovnkube.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: env-overrides
+  namespace: openshift-ovn-kubernetes
+data:
+  x86-worker-node0: |
+    OVNKUBE_NODE_MGMT_PORT_NETDEV=ens1f0v0
```

Comment:
ens1f0v0 is the VF name that is assigned to the ovnkube node management port on the host.